KubeArmor/build/kubearmor-test-containerd.yaml

```diff
@@ -89,6 +89,12 @@ spec:
         - -gRPC=32767
         - -logPath=/tmp/kubearmor.log
         - -enableKubeArmorHostPolicy
+        - -defaultFilePosture block
+        - -defaultCapabilitiesPosture block
+        - -defaultNetworkPosture block
+        - -hostDefaultCapabilitiesPosture block
+        - -hostDefaultNetworkPosture block
+        - -hostDefaultFilePosture block
```

Comment:
Have you tried the image and YAML locally?

![image](https://user-images.githubusercontent.com/47106543/201908622-dbeb8b86-1467-40b9-80e9-f3c2eb140044.png)

I think they need to be this
```suggestion
        - -defaultFilePosture=block
        - -defaultCapabilitiesPosture=block
        - -defaultNetworkPosture=block
        - -hostDefaultCapabilitiesPosture=block
        - -hostDefaultNetworkPosture=block
        - -hostDefaultFilePosture=block
```